master/master.cfg

```diff
@@ -73,6 +73,35 @@ def do_step_build_spl(step):
 def do_step_build_zfs(step):
     return do_step_build(step, 'buildzfs')
 
+style_factory = util.BuildFactory()
+
+style_factory.addStep(Git(repourl=zfs_repo, workdir="build/zfs",
+    mode="full", method="clobber", codebase="zfs",
+    logEnviron=False, getDescription=True,
+    description=["cloning"], descriptionDone=["cloned"]))
+style_factory.addStep(ShellCommand(command=["./autogen.sh"],
+    env={'PATH' : bin_path},
+    workdir="build/zfs", logEnviron=False,
+    decodeRC={0 : SUCCESS, 1 : FAILURE, 2 : WARNINGS, 3 : SKIPPED },
+    description=["autogen"], descriptionDone=["autogen"]))
+style_factory.addStep(ShellCommand(command=["./configure"],
```

Comment:
Without the SPL installed this is going to fail.  But we could speed this up and remove the SPL dependency by running `./configure --with-config=srpm`.  This disables most of the configure checks and allows us to use some make targets like `dist`, `cstyle`, etc.  Anything which doesn't actually requires us to compile the code.